requirements.txt

```diff
@@ -21,5 +21,5 @@ plotly
 tqdm
 black
 pre-commit
-fsspec
+fsspec==2021.7.0
```

Comment:
Ooh, interesting, are we sure we can't use more recent versions of `fsspec`?  A few days ago, there was a bug (I think) in fsspec which forced pip to install an ancient version of `gscfs` but that appears fixed now (if that's what make you pin `fsspec` to version 2021.7.0?